tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -109,317 +49,306 @@ describe('Target aggregates', () => {
       expect((await targetAggregatesPage.aggregateList()).length).to.equal(0);
-      expect(
-        await (await analyticsPage.emptySelectionError()).isDisplayed()
-      ).to.be.true;
+      expect(await analyticsPage.emptySelectionError().isDisplayed()).to.be.true;
     });
   });
 
-  describe('as a user with a home place', () => {
-    const parentPlace = placeFactory.place().build({ type: 'district_hospital' });
-    const otherParentPlace = placeFactory.place().build({ type: 'district_hospital' });
-    const user = userFactory.build({ place: parentPlace._id, roles: ['program_officer'] });
-    const names = ['Clarissa', 'Prometheus', 'Alabama', 'Jasmine', 'Danielle'];
-    const otherNames = ['Viviana', 'Ximena', 'Esteban', 'Luis', 'Marta'];
-    const generatePlace = (parent, idx, otherPlace) => {
-      const place = placeFactory.place().build({ type: 'health_center', parent: { _id: parent._id } });
-      const contact = personFactory.build({
-        name: otherPlace === true ? otherNames[idx] : names[idx],
-        parent: { _id: place._id, parent: place.parent }
-      });
-      place.contact = { _id: contact._id, parent: contact.parent };
-      return [place, contact];
-    };
-    const docs = _.flattenDeep([
-      Array.from({ length: 5 }).map((e, i) => generatePlace(parentPlace, i, false)),
-      Array.from({ length: 5 }).map((e, i) => generatePlace(otherParentPlace, i, true)),
-    ]);
-    const generateTitle = (title) => ({ en: title });
-    const docTags = [
-      // current targets
-      moment().format('YYYY-MM'),
-      // next month targets, in case the reporting period switches mid-test
-      moment().date(10).add(1, 'month').format('YYYY-MM'),
-    ];
+  describe('User with one or more places assigned', () => {
+    const CURRENT_PERIOD = 'This month';
+    const NAMES_DH1 = ['Clarissa', 'Prometheus', 'Alabama', 'Jasmine', 'Danielle'];
+    const NAMES_DH2 = ['Viviana', 'Ximena', 'Esteban', 'Luis', 'Marta'];
+
+    const TARGET_VALUES_BY_CONTACT = helperFunctions.generateTargetValuesByContact([...NAMES_DH1, ...NAMES_DH2]);
+    
+    const districtHospital1 = placeFactory.place().build({ type: 'district_hospital', name: 'District Hospital 1' });
+    const districtHospital2 = placeFactory.place().build({ type: 'district_hospital', name: 'District Hospital 2' });
+
     const contactWithManyPlaces = personFactory.build({
-      parent: { _id: parentPlace._id, parent: { _id: parentPlace._id } },
+      parent: { _id: districtHospital1._id, parent: { _id: districtHospital1._id } },
     });
-    const userWithManyPlaces = {
+
+    const onlineUser = userFactory.build({ place: districtHospital1._id, roles: ['program_officer'] });
+    
+    const userWithManyPlaces = userSettingsFactory.build({
       _id: 'org.couchdb.user:offline_many_facilities',
-      language: 'en',
-      known: true,
-      type: 'user-settings',
+      id: 'org.couchdb.user:offline_many_facilities',
```

Comment:
This `id` has the same value as `_id`. Can it be removed?

Reply:
Yeap. Removed 😄